learn/getting-started/create-your-first-application.mdx

```diff
@@ -0,0 +1,340 @@
+
+With Harper successfully installed and setup, let's dive into building your first Harper Application, a simple REST API. Harper lets you build powerful APIs with minimal effort.
+
+## What You Will Learn
+
+- Overview of Harper architecture
+- What are Harper core services, plugins, and applications
+- How to define a table using schemas
+- How to run a Harper application
+- How to automatically create a REST API from a table schema
+- How to interact with the table using the generated REST API
+
+## Prerequisites
+
+- Working Harper Installation (previous guide [Install and Connect Harper](./install-and-connect-harper))
+
+## Harper Architecture
+
+Before diving into building your first Harper application, it is important to understand a bit about Harper's architecture. The simplest way to think about Harper is as a stack.
+
+```
+┏━━━━━━━━━━━━━━━━━━┓
+┃ Applications     ┃
+┠──────────────────┨
+┃ Plugins          ┃
+┃  - rest          ┃
+┃  - graphqlSchema ┃
+┃  - ...           ┃
+┠──────────────────┨
+┃ Core Services:   ┃
+┃  - database      ┃
+┃  - networking    ┃
+┃  - component     ┃
+┃    management    ┃
+┗━━━━━━━━━━━━━━━━━━┛
+```
+
+At the bottom are the **core services** that make up the foundation of the Harper platform. This includes the high-performance **database**, extensible **networking** middleware, and **component** management system. Components are extensions of the core Harper system, and are further classified as **plugins** and **applications**.
+
+**Plugins** come next in the stack. Plugins have access to APIs exposing many of Harper's core services, and are capable of implementing more advanced features than what the core services provide. Many of Harper's features are implemented as **built-in plugins**. Additional features can be implemented as **custom plugins**. In this guide, we'll be demonstrating some of Harper's built-in plugins `graphqlSchema` and `rest`. Later guides will demonstrate many more.
+
+And finally, the top of the stack are **applications**. This is where any user-facing functionality is implemented. Applications use plugins to implement their business logic. Everything from database table schemas, to web applications.
+
+The key difference between plugins and applications is that plugins enable the functionality, and applications implement it. Its similar to that of a front-end framework. React on its own doesn't actually do anything, you actually need to build a "React App" for it to do anything meaningful.
+
+## Initializing the Harper Application
+
+Lets get started building your first Harper application!
+
+<Tabs groupId="installation">
+	<TabItem value="local" label="Local Installation">
+
+Get started by cloning the [`HarperFast/create-your-first-application`](https://github.com/HarperFast/create-your-first-application) repo and opening it your editor of choice. If you have installed Harper using a container, make sure to clone into the `dev/` directory that the container was mounted to.
```

Comment:
I was under the impression that our goal was that the documentation narrative would start with `HarperFast/application-template` and ultimately end up at `HarperFast/application-example`. Or something like that. And I think SG had asked that we tell people the repo of the example app that we are building towards, right from the start (which I agree with). But maybe that just isn't ready yet?